File: Server/JobModule/Models/CurrentJobInfo.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace JobModule.Models
{
    public class CurrentJobInfo
    {
        public DateTime DeliveryTime { get; set; }
        public uint PlanedDistance { get; set; }
        public bool IsLate { get; set; }
        public bool IsLoaded { get; set; }
        public DateTime RemainingDeliveryTime { get; set; }
        public ulong Income { get; set; }

        public Point Source {  get; set; }
        public Point Destination { get; set; }
        public Cargo Cargo { get; set; }
    }
}

File: Client/app/src/main/java/com/nks/interactive/multimediapanel/models/job/CurrentJobInfo.kt
package com.nks.interactive.multimediapanel.models.job

import com.nks.interactive.multimediapanel.models.job.Cargo
import com.nks.interactive.multimediapanel.models.job.Point
import java.time.LocalDateTime

data class CurrentJobInfo(
    val deliveryTime: LocalDateTime,
    val planedDistance: Int,
    val isLate: Boolean,
    val isLoaded: Boolean,
    val remainingDeliveryTime: LocalDateTime,
    val income: Long,
    val source: Point,
    val destination: Point,
    val cargo: Cargo
)